draft-ietf-quic-multipath.md

```diff
@@ -1098,7 +1098,8 @@ Handshake and Application packets).
 
 For any given path, connection ID rotation, NAT rebinding, or client initiated migration
 as specified in {{QUIC-TRANSPORT}} might occur, like on a single path.
-These events do not change the path ID,  and do not affect the packet number
+These events do not change the path ID,
+ and do not affect the packet number
```

Comment:
I am not sure if this was an unintentional change (this does not change anything except adding a new line?).
```suggestion
These events do not change the path ID, and do not affect the packet number
```